shells/lib/env/env-base.js

```diff
@@ -28,9 +32,8 @@ export class EnvBase {
       'https://$artifacts/': `${root}/particles/`, // deprecated
     };
   }
-  // loader construction is lazy, so pathMap can be configured prior
```

Comment:
Laziness was part of the contract of this object, we can't just eliminate it without taking care.

Reply:
Yes, but we looked at it and found the only use to be the call immediately preceding, in the constructor, which doesn't make use of the laziness. Did we miss another call?